opentelemetry-api/src/opentelemetry/util/__init__.py

```diff
@@ -48,3 +49,42 @@ def _load_provider(provider: str) -> Union["TracerProvider", "MeterProvider"]:
             "Failed to load configured provider %s", provider,
         )
         raise
+
+# Pattern for matching the 'https://', 'http://', 'ftp://' part.
+URL_PATTERN = '^(https?|ftp):\\/\\/'
+
+
+def disable_tracing_url(url, blacklist_paths):
+    """Disable tracing on the provided blacklist paths
+
+    If the url path starts with the blacklisted path, return True.
+
+    :type blacklist_paths: list
+    :param blacklist_paths: Paths to prevent from tracing
+
+    :rtype: bool
+    :returns: True if not tracing, False if tracing
+    """
```

Comment:
```suggestion
def disable_tracing_url(url, excluded_paths):
    """Disable tracing on the provided excluded paths

    If the url path starts with the excluded path, return True.

    :type excluded_paths: list
    :param excluded_paths: Paths to exclude from tracing

    :rtype: bool
    :returns: True if not tracing, False if tracing
    """
```

Reply:
In the code above, the parameter says path, but the method says url, my suggestion is to change this to make it consistent